MovieDB/src/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Search } from './components/search-component';
+import MovieTileItem from './components/movie-tile-item';
```

Comment:
Please be consistent. 
1. Imports. One component is imported as default, another not. Both approaches are fine in general, just choose the one and use it. More right might be to use `default` approach as far `component` - is what you expect to get from a component file.
2. Naming. `movie-tile-item`, `search-component`. If you want to add `component` postfix - it's fine, but then add it to all components and filenames. Or just remove from `search` :)

Reply:
Fixed!